docs/index.rst

```diff
@@ -31,6 +31,10 @@ that a release will be instantaniously available after each set of changes
 made to the code.
 The latest version of astroquery can be pip installed.
 
+\*Note that if an environment already has a release version of astroquery installed
+when this command is run, it will not install the latest release.
+In this case astroquery will need to be uninstalled before the pip command is run.
```

Comment:
Perhaps add, `(e.g., by running ``pip uninstall astroquery``)`, if that's what you did?

Reply:
or add ``--upgrade``

Reply:
@bsipocz  Oh that's even easier. Yay. (updated it)